py_irt/models/four_param_logistic.py

```diff
@@ -199,6 +201,18 @@ def export(self):
             "lambdas": pyro.param("lambdas").data.tolist(),
         }
 
+    def predict(self, models, items, params_from_file=None):
```

Comment:
I think we should be consistent and use `subjects` here

Reply:
Agree, updated throughout.